rachelnavarro/step/portfolio/src/main/webapp/script.js

```diff
@@ -51,3 +51,32 @@ function getDateTime() {
   var datetime = `${today.getFullYear()}-${today.getMonth()+1}-${today.getDate()} ${today.getHours()}:${today.getMinutes()}:${today.getSeconds()}`;
   return datetime;
 }
+
+/** Creates and loads maps for all the locations featured in places.html. 
+ * TODO: fetch this info from a java servlet instead of hardcoding */
+function loadMaps() {
+  var placeCoordinates = [
+    [40.704381, -73.990588], // DUMBO Waterfront- Brooklyn, New York
```

Comment:
I think would be cleaner if you used a kv map where the keys are things like "nyc" and "tokyo" and the values are these latitude longitude pairs. In javascript you could say like

```
var places = {
  "nyc": [40.8, -73.9],
...
};
```